docs/source/api.rst

```diff
@@ -1,15 +1,35 @@
-Unstructured API
-#################
+Unstructured API Services
+#########################
 
-Try our hosted API! You can get your API key `here <https://unstructured.io/api-key>`__ now and start using it today.
+The Unstructured API presents an extensive selection of API services, encompassing free access, Software-as-a-Service (SaaS), and Azure & AWS Marketplace integrations. These API services offer a high-quality and scalable solution capable of ingesting and digesting files of various types and sizes.
 
-You will find a more comprehensive overview of the API capabilities. For detailed information on request and response schemas, refer to the `API documentation <https://api.unstructured.io/general/docs#/>`__.
+Within this documentation, you will find a detailed guide for implementing the Unstructured API across these diverse service options to best meet your operational requirements. It includes details on supported file types, step-by-step guides on using various API services, Python and JavaScript SDKs, and API parameters & common error handling.
 
-NOTE: You can also host the API locally. For more information check the `Using the API Locally <https://github.com/Unstructured-IO/unstructured-api>`__ section.
+.. note::
+    Please review this manual carefully, as we've introduced significant updates to the API services, including a new usage cap for our free API tier, effective from January 10th, 2024.
 
 
+Changes to the Free API Service
+*******************************
+
+Implementation of a 1000-Page Cap
+---------------------------------
+Starting from January 10th, 2024, our free API service includes a cap of 1,000 pages. This limitation is introduced to ensure the sustainability of our free offerings and to enhance service quality.
+
+.. warning::
```

Comment:
Also add that documents are never stored on the paid API, and that users can switch to that if they don't want documents stored.

Reply:
I know you allude to that in the section below, but just want to make sure we're double tapping that